css-layout-classes/styles.css

```diff
@@ -0,0 +1,30 @@
+.container {
+  margin: 0 auto;
+  width: 800px;
+}
+
+h1 {
+  font-family: Arial, sans-serif;
+}
+
+.row {
+  display: flex;
+}
+
+.column-fourth {
+  flex-basis: calc(100% /4);
+}
+
+
```

Comment:
Once you adjust your class names, you'll need to adjust the css. You should have more column selectors here. Compare against the solution I posted in the main class channel